analyses/tcd/metrics/TCD_trigger_report.Rmd

```diff
@@ -0,0 +1,432 @@
+explanatory_ci
+```
+
+* <p style="color:black;font-size:16px;text-align:left;"> The <span style="font-weight: bold;">central value</span> is the most likely performance; </p>
+* <p style="color:black;font-size:16px;text-align:left;"> Most often the trigger will perform within the <span style="font-weight: bold;">darker inner range</span> (68% of the time);</p>
+* <p style="color:black;font-size:16px;text-align:left;"> There is high confidence that the trigger will perform within the <span style="font-weight: bold;">full coloured (grey or blue) range</span> (95% of the time).</p> 
+
+<br>
+
+## Colour Coding
+<p style="color:black;font-size:16px;text-align:left;"> In the visualizations <span style="color: blue;font-weight: bold;">blue</span> represents activation likelihood, <span style="color: green;font-weight: bold;">green</span> represents desired outcomes (valid non-activations/activations) and <span style="color: red;font-weight: bold;"dou>red</span> represents undesired outcomes or errors.</p>
+
+&nbsp;
+
+## Trigger Mechanism Snapshot
+<p style="color:black;font-size:16px;text-align:left;"> This table summarises the trigger mechanism and its performance in signaling severe shocks. It presents estimates of the likelihood that the threshold would be met in a given year as well as how often the trigger is expected to correctly recommend an activation or non-activation.</p> 
+
+&nbsp;
+```{r mechanism-snapshot, include = TRUE}
+
+snapshot_df <- data.frame(row_name = c('Type', 
+                                       'Monitored Area',
+                                       'Activation Timepoints', 
+                                       'Target Period', 
+                                       'Lead Time', 
+                                       'Data Source',
+                                       'Trigger',
+                                       'Historical Data Coverage',
+                                       'Monitored by',
+                                       'Activity Package', 
+                                       'Probability of being met in a given year', 
+                                       'Funding Amount',
+                                       'Expected Annual Cost',
+                                       'All Activations (%)', 
+                                       'All Shocks (%)'),
+                       trigger_a = c(type_a, 
+                                     monitored_area_a,
+                                     timepoints_a, 
+                                     targeted_window_a, 
+                                     lead_time_a, 
+                                     data_source_a,
+                                     trigger_a,
+                                     hist_period_a,
+                                     monitored_by_a,
+                                     package_a, 
+                                     paste0(prob_a, "%, based on global IRI data, but very high uncertainty"),
```

Comment:
The pilot team & HQ likely expect something around 35%. This is particularly low. Perhaps worth a separate footnote to elaborate?